client/src/style/header.css

```diff
@@ -0,0 +1,32 @@
+header{
```

Comment:
This CSS should live next to Header.js in a file called Header.css - and it should be only imported in Header component